frontend/src/components/Login/Login.css

```diff
@@ -1,13 +1,15 @@
 .buttonStack {
-    position: absolute;
+    position: relative;
+    left: 770px;
+    bottom: 110px;
```

Comment:
These pixel values are way too high. This is not responsive when the size of the screen changes. It'll only look right on your screen.